xcp_d/interfaces/c3.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this already in nipype? If it is, we should probably import it instead of copying the interface here

Reply:
The "multicomp_split" trait in Nipype's c3d interface is incorrect (at least for the version of c3d I put in xcpd_build), so I made this modified copy of the interface for now